### ISSUE TITLE
2a- vi) description or visualization 

### ISSUE THREAD

**Issue by miamacarena** [iiiMatt/WiSe-2023-24#11]
We can and will create another function and we are free to decide what it aimed to do. 

Let us assign it to one of us four (Mate, Loic, Johanna, Mia) and brainstorm here which function would help us in excercise 4.

**Comment by Johanna2208** [comment]
I would graphically represent age in relation to survival. This could be interesting.

**Comment by miamacarena** [comment]
I think it is rather about the graphics: mosicplot etc. so that the functions can be used with suiting variables. 
Which variables are included exactly is decided in excercise 4) as far as i understand. 
But the idea is good: just prep functions that can be used like that when we proceed with 4)

**Comment by Johanna2208** [comment]
yes did that, i wrote one function for mosaicplots and one for boxplots

**Comment by miamacarena** [comment]
Perfect!
